weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/EmptyContent.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import {TargetBlank} from '../../../../../../common/util/links';
+import {Link, TargetBlank} from '../../../../../../common/util/links';
```

Comment:
Looks like this Link doesn't have the standard styling, maybe you want https://github.com/wandb/weave/blob/jamie/feedback-schema/weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/Links.tsx#L37

Reply:
fixed! thanks